todo_app/view_model.py

```diff
@@ -0,0 +1,31 @@
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def done_items(self):
+        done_list = []
+        for item in self.items:
+            if item.status == 'Done':
+                done_list.append(item)
+        return done_list
```

Comment:
Not one I necessarily recommend, but I always think it's worth seeing a comparison with the concise (but arguably less readable) one-liner list comprehension approach here:
```
return [item for item in self.items if item.status == 'Done']
```

Reply:
Thanks for suggesting it, I have refactored view model to use list comprehension throughout